### PR TITLE
Anime List generated by BeautifulSoup 

### PR DESCRIPTION
**Pull request by infraSamurai**
Fetched and scraped data from [myanimelist.net/anime/(some value)/] from BeautifulSoup and request module. The pages was quite a large in number, so, having a low internet speed I collected data from 20 pages and it goes very well for more. Also, handled the exception when the page is not found which was quite frequent. Collected the data such as Name of the Anime, Rating, Description, Ranking and Number of episodes.
Also, stored the data in anime_scrape.json file to be used furthur.

### CODE REVIEW

**Comment by TheLethalCode** [comment]
This is good, but you are running your code in python2, but we are using python3. Would be better if you make the code python3 compatible. One more thing, though you have used functions, your for loop is still in the global space. Place the 'for' loop under the main, and leave the info_anime in the global namespace.

**Comment by infraSamurai** [comment]
The suggested changes are made. Ready to be merged.

File: myanimelist.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+	print (ep.text)
+
+
+if __name__ == '__main__':
+	'''
+	Iterating the loop from 1 to 30000 to store data from myanimelist
+	where the url is  https://myanimelist.net/anime/(some number)/
+	'''
+
+for index in range(1,30000):
```
Comment (TheLethalCode) [inline myanimelist.py]: Indentation required

Comment (CircleJerkHug) [reply myanimelist.py]: screenshot from 2018-12-09 16-56-22
I don't see any indentation issue in my code, rather the code in github is showing up differently.

**Comment by TheLethalCode** [comment]
Hey, you have sent the PR to the wrong branch. Resend it to the devel branch

**Pull request closed without merging**